frontend/projects/upgrade/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.html

```diff
@@ -116,7 +116,7 @@
     <div class="chart">
       <div class="chart-data">
         <ngx-charts-bar-vertical-stacked
-          [scheme]="colorScheme"
+          [scheme] = "colorScheme"
```

Comment:
remove this spaces around `=` for consistency with other params

Reply:
Removed this whitespace for consistent code